.cirrus.yml

```diff
@@ -2,15 +2,25 @@ env:
   ELIXIR_ASSERT_TIMEOUT: 2000
   LANG: C.UTF-8
 
-test_task:
+test_linux_task:
+  name: Test (Linux - Ubuntu 14.04) / ${OTP_RELEASE}
```

Comment:
```suggestion
  name: Test ${OTP_RELEASE} on Linux (Ubuntu 14.04)
```

The main goal is to have the version of the OTP release in the sidebar:

![Screen Shot 2019-11-21 at 09 46 45](https://user-images.githubusercontent.com/651203/69321710-dbe85500-0c43-11ea-9d62-f3510217926d.png)

Reply:
Excelent comment. Please have a look now